packages/dashboard-core-plugins/src/panels/FilterSetManagerPanel.test.tsx

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+import { FilterSetManagerPanel } from './FilterSetManagerPanel';
+
+function makeGlComponent() {
+  return {
+    on: jest.fn(),
+    off: jest.fn(),
+    emit: jest.fn(),
+    tab: null,
+  };
+}
+
+function makeColumns(count = 30) {
+  const columns = [];
+  for (let i = 0; i < count; i += 1) {
+    const column = { index: i, name: `${i}` };
+    columns.push(column);
+  }
+  return columns;
+}
+
+function renderFilterSetManagerPanel({
+  filterSets = [],
+  localId = 'TEST_ID',
+  panelMap = new Map(),
+  tableMap = new Map(),
+  setFilterSets = jest.fn(),
+} = {}) {
+  render(
```

Comment:
You can do `return render`, then use `getByText`, instead of `screen.getByText`

Reply:
Good point, updated!